ppx_irmin.opam

```diff
@@ -0,0 +1,24 @@
+opam-version: "2.0"
+maintainer: "Craig Ferguson <craig@tarides.com>"
+author: "Craig Ferguson <craig@tarides.com>"
+homepage: "https://github.com/CraigFe/ppx_irmin"
+bug-reports: "https://github.com/CraigFe/ppx_irmin/issues"
+license: "BSD-2"
+dev-repo: "git+https://github.com/CraigFe/ppx_irmin.git"
+
+build: [
+  [ "dune" "build" "-p" name "-j" jobs ]
+]
+
+run-test: [
+  [ "dune" "runtest" "-p" name "-j" jobs ]
```

Comment:
The other packages have [these test runes](https://github.com/mirage/irmin/blob/66bcb96e682d67db837c958a48921c62de015fa4/irmin.opam#L10-L14) instead.